.github/workflows/publish.yml

```diff
@@ -98,5 +104,6 @@ jobs:
           GH_TOKEN: ${{ github.token }}
         working-directory: build
         run: |
-          Compress-Archive -Path "bmx" -DestinationPath "bmx-$env:PLATFORM.zip"
-          gh release upload "$env:RELEASE_TAG" "bmx-$env:PLATFORM.zip"
+          chmod +x "bmx"
+          tar -czvf "bmx-$PLATFORM.tar.gz" "bmx"
+          gh release upload "$RELEASE_TAG" "bmx-$PLATFORM.tar.gz"
```

Comment:
need to change `shell: ` to really change the shell
https://github.com/Brightspace/bmx/pull/271/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R100